clients/desktop/src/vault/qr/upload/UploadQrPage/UploadQrPageWithExistingVault.tsx

```
@@ -0,0 +1,73 @@
+import { useAppNavigate } from '../../../../navigation/hooks/useAppNavigate'
+import { useAppPathParams } from '../../../../navigation/hooks/useAppPathParams'
+import { useNavigateBack } from '../../../../navigation/hooks/useNavigationBack'
+import { FlowPageHeader } from '../../../../ui/flow/FlowPageHeader'
+import { useCurrentVaultCoins } from '../../../state/currentVault'
+import { ScanQrView } from '../ScanQrView'
+import { UploadQrView } from '../UploadQrView'
+import { useDeriveChainFromWalletAddress } from '../useDeriveChainFromWalletAddress'
+import { StyledPageContent } from './UploadQRPage.styled'
+
+const uploadQrViews = ['scan', 'upload'] as const
+type UploadQrView = (typeof uploadQrViews)[number]
```

Biome (1.9.4)

[error] Shouldn't redeclare 'UploadQrView'. Consider to delete it or rename it.

'UploadQrView' is defined here:


(lint/suspicious/noRedeclare)